src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -498,6 +498,22 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val verboseAuditTarget: PipelineTable = PipelineTable(
+      name = "verboseAuditTarget_gold",
+      _keys = Array("organization_id", "timestamp", "unixTimeMS"),
+      config,
+      _mode = WriteMode.merge,
+      partitionBy = Seq("organization_id"),
+      incrementalColumns = Array("unixTimeMS"),
+      zOrderBy = Array("notebookId", "unixTimeMS")
```

Comment:
zorders are expensive and I don't really see many people (or downstream ETLs) referencing notebookId or unixTimeMS heavily, do you? Zorders are meant to speed up ETL and/or end-user queries -- I foresee people filtering on notebookPath and using wildcards and/or the actual timestamp column much more than unixTimeMS. 

If you agree -- I suggest removing these orders -- if you disagree, please make your case and we can review.